src/SharpLouis/SharpLouis.csproj

```
@@ -28,12 +28,16 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <Content Include="LibLouis/**">
+        <!-- Content is included in packages by default. Much more detail on packing can be found at
+             https://learn.microsoft.com/en-us/nuget/reference/msbuild-targets -->
+        <Content Include="LibLouis\**" Exclude="LibLouis\liblouis.dll">
             <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-            <Pack>true</Pack>
             <Visible>false</Visible>
-            <PackagePath>lib/$(TargetFramework)/LibLouis</PackagePath>
+            <!-- But by default the Content won't be included by projects that reference your package. This metadata
+                 changes that default behavior.  -->
+            <PackageCopyToOutput>true</PackageCopyToOutput>

```

If the `LibLouis` prefix on the final published file paths is harmful, we can remove it by applying the following explicit guidance to NuGet about the file paths we want:

```suggestion
            <PackageCopyToOutput>true</PackageCopyToOutput>
            <PackagePath>contentFiles/any/$(TargetFramework)</PackagePath>
```

This will cause the files to change layout from `contentFiles/any/net8.0/LibLouis/tables.json` before `contentFiles/any/net8.0/tables.json` after, for example.